packages/inventory/src/services/inventory.ts

```diff
@@ -79,7 +83,10 @@ export default class InventoryService implements IInventoryService {
    * @param config - the find configuration to use
    * @return A tuple of inventory levels and their total count
    */
-  @InjectEntityManager()
+  @InjectEntityManager(
+    (target) =>
+      target.moduleDeclaration?.resources === MODULE_RESOURCE_TYPE.ISOLATED
```

Comment:
suggestion(non-blocking): create a method with a descriptive name for this method to just pass `moduleIsIsolated` or something similar